03 Misc/01 Linting/README.md

```diff
@@ -58,9 +60,9 @@ We are going to create a file `.eslintrc.json` (there are many [file formats opt
 {
   ...
   "scripts": {
--   "start": "webpack-dev-server"
-+   "start": "webpack-dev-server",
```

Comment:
These lines are to note the reader: "hey, don't forget the comma", what do you think if we keep it?